source/agent/conference/roomController.js

```diff
@@ -903,7 +905,8 @@ module.exports.create = function (spec, on_init_ok, on_init_failed) {
     var getAudioTranscoder = function (stream_id, on_ok, on_error) {
         findExistingAudioTranscoder(stream_id, on_ok, function () {
             var axcoder = randomId();
-            newTerminal(axcoder, 'axcoder', streams[stream_id].owner, false, function () {
+            var terminalid = streams[stream_id].owner;
```

Comment:
`terminalid` is not consistent with `terminal_id` or `terminalId` in other places